### ISSUE TITLE
Bug in supervisor_emiter_receiver.py

### ISSUE THREAD

**Issue by DreamtaleCore** [aidudezzz/deepbots#20]
💗💗It's a nice job! 
However, I find there exists a tiny 👾 in `supervisor_emiter_receiver.py`, L-77, that occurs when the receiver gets nothing from the queue, the `_last_message`  may destroy the logic of the supervisor. In detail, [deepbots-tutorials](https://github.com/aidudezzz/deepbots-tutorials/tree/master/cartPoleTutorial) will not work because of this issue.

So I modified the function below:
```python
def handle_receiver(self):
        if self.receiver.getQueueLength() > 0:
            string_message = self.receiver.getData().decode("utf-8")
            self._last_message = string_message.split(",")

            self.receiver.nextPacket()

        return self._last_message
```
👉
``` python
def handle_receiver(self):
        if self.receiver.getQueueLength() > 0:
            string_message = self.receiver.getData().decode("utf-8")
            self._last_message = string_message.split(",")

            self.receiver.nextPacket()
        else:
            self._last_message = None

        return self._last_message
```

then everything works smoothly for me 😃.

**Comment by tsampazk** [comment]
Hello @DreamtaleCore, thank you very much for your feedback! 

This issue is actually related to #15. You are right that this causes the tutorial world to crash due to receiving `done = True` even after the world is reset and a new episode begins.

This happens because the supervisor has saved the last message of the previous episode and did not reset it when the episode ended. At the start of the new episode the supervisor does not receive any message so the old one gets returned. 

Your fix works well for now, it will be fixed soon.

Any other issues or PRs are welcome. :)

**Comment by tsampazk** [comment]
The issue is fixed, changes are included in latest release (0.1.0.1)

**Comment by tsampazk** [comment]
@all-contributors please add @DreamtaleCore  for bug
